log-collector/src/main/scala/io/phdata/pulse/logcollector/PulseKafkaConsumer.scala

```diff
@@ -55,7 +55,7 @@ class PulseKafkaConsumer(solrService: SolrService) extends JsonSupport with Lazy
           logger.trace("KAFKA: Consuming " + record.value() + " from topic: " + topic)
           val logEvent    = record.value().parseJson.convertTo[LogEvent]
           val logEventMap = Util.logEventToFlattenedMap(logEvent)
-          solrInputStream ! (logEventMap.getOrElse("application", None), logEventMap)
+          solrInputStream.put(logEventMap.getOrElse("application", ""), logEventMap)
```

Comment:
Does it make sense to have an application name as `""`? Or should this be an error?

Reply:
No it doesn't.  I thought about what to do with this, it should either be just thrown away or logged. We want to figure out how to debounce logging before we do that though. Also only way this code path can be reached is through the Kafka appender, so it's not an immediate concern. I'll add a task for a future commit to fix this.

Reply:
@safwanislam can you add a TODO comment in your next commit to address this issue?